### ISSUE TITLE
Virus , mod damages your pc.

### ISSUE THREAD

**Issue by Reraz** [hydy100/R3nzSkin#227]
After i had this mod mi pc started to have random blue screen, i didnt think it was this mod so i reinstalled windows, the same day i reinstalled windows and decided to use this mod the bllue screen appeared making my pc unusable, after reinstalling windows again i decided to stop using this mod and now its been days and ive had no blue screen sincde then.

**Comment by Asyozu** [comment]
why would someone want to put a virus in open source code?

**Comment by Reraz** [comment]
> why would someone want to put a virus in open source code?I dont know though the only change i did was no using the mod or download it  and my pc is fine.

**Comment by mish20011** [comment]
> After i had this mod mi pc started to have random blue screen, i didnt think it was this mod so i reinstalled windows, the same day i reinstalled windows and decided to use this mod the bllue screen appeared making my pc unusable, after reinstalling windows again i decided to stop using this mod and now its been days and ive had no blue screen sincde then.

yeah sure bud, I've been using this since vanguard came out and never had a single issue

**Comment by VERSIONADC** [comment]
That's nice my pc got system files deleted because this program :)
